src/annotator/bucket-bar.js

```diff
@@ -5,7 +5,8 @@ import { anchorBuckets } from './util/buckets';
 
 /**
  * @typedef BucketBarOptions
- * @prop {string[]} [scrollables] - Selectors for the scrollable elements on the page
+ * @prop {Element} [contentContainer] - The scrollable container element for the
```

Comment:
I would suggest `scrollableContainer`

Reply:
or better `scrollableElement`

Reply:
`contentContainer` is ambiguous. It seems to refer to the container of bucket-bar itself.

Reply:
`scrollableContainer` is fine with me.

Reply:
Hmm ... I'm having second thoughts about this. `scrollableContainer` doesn't really capture what I mean. This element is really the container for the _document content_ (hence the original name) and all of the highlights. "content" is short for "document content". In future we might want to observe other kinds of events on this element - such as resizing which might alter the layout.